CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Welcome! 🎉
+
+Welcome to the Electricity Maps open source contribution repository. </br>
+Any and all contributions however big or small are welcome.
+
+# Contribution Guidelines
+
+## Non-code contributions
+
+There are several ways to help out without coding, these are primarily:
+
+- Opening issues for bugs.
+- Opening issues for data problems.
+- Opening and/or participating in discussions about new data sources, features, and more.
+- Opening issues when capacity data sources have been updated or changed.
+- And more!
+
+## Parser guidelines
+
+To get stated with editing the parsers use the following steps:
+
+1. Run `poetry install -E parsers` to install all needed dependencies.
+2. Use `poetry run test_parser ZONE_KEY` to test any parser changes.
+
+Note: This requires you to have [Python][python homepage] and [Poetry][poetry homepage] installed, you can see their respective installation guides here:
+
+- [Downloading Python][python install guide]
+- [poetry installation][poetry install guide]
+
+### Parser requirements
+
+There are no clear cut model or template that works for all parsers as all data are different and unique but there are some basic requirements all parsers need to follow:
+
+- They need to return valid data from a credible source.
+- They need to return the date for when the data was collected together with the data.
+- They need to return at least one valid Electricity Maps data object (production, consumption, exchange, price, etc).
```

Comment:
Maybe an example blob would be nice

Reply:
Given your recent work on the parsers/examples, I would actually suggest removing the example here and pointing to that folder - that way there's fewer places where things might get out of sync of something changes  :)